src/ripples/touch-ripple.jsx

```diff
@@ -119,6 +119,7 @@ const TouchRipple = React.createClass({
   },
 
   _handleTouchStart(event) {
+    event.stopPropagation();
```

Comment:
you've based your work on a commit that's not on the master branch. usually happens when you checkout a new branch without checking out master first. Please do: 

```
git rebase -i master
```

and then only pick this commit removing the rest. thanks :grin: